app/controllers/staff/grids_controller.rb

```diff
@@ -2,10 +2,10 @@ class Staff::GridsController < Staff::ApplicationController
   include ScheduleSupport
 
   def show
-    @schedule = Schedule.new(current_event)
-    @counts = EventStats.new(current_event).schedule_counts
-    @sessions = current_event.program_sessions
-    @unscheduled_sessions = @sessions.unscheduled
-    @tracks = current_event.tracks.sort_by_name
+    # @schedule = Schedule.new(current_event)
```

Comment:
If we're not planning to use these, let's delete the lines.  We can always use git to gain access if we need them for some reason.   Better to remove than leave them commented out for future devs to wonder.

Reply:
Thank you for this, addressed in this push I am deploying.